client/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Platform, StyleSheet, Text, View } from 'react-native';
+
+import Card from './js/screens/Card';
+// import RootStackNavigator from './navigation/RootStackNavigator';
+
+const instructions = Platform.select({
```

Comment:
please remove unnecessary code and imported modules.

Reply:
I have removed the unnecessary code and imported modules.